kubernetes/samples/scripts/create-weblogic-domain/domain-home-on-pv/wlst/create-domain.py

```diff
@@ -62,11 +59,6 @@ def getEnvVar(var):
 set('ListenAddress', '%s-%s' % (domain_uid, admin_server_name_svc))
 set('ListenPort', t3_channel_port)
```

Comment:
- We should be able to remove the ListenAddress  for admin server, etc
- Need to check if sit-config is overriding ListenAddress for T3Channel

Reply:
These bullets already have follow-on JIRAs.